### PR TITLE
Docs: additional info about address spaces

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/con-standard-address-space.adoc [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 [id='con-standard-address-space-{context}']
 = Standard Address Space
-The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, it has the following limitations: 
+The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, the standard address space has the following limitations:
 
 * No transaction support
 * No message ordering
-* No selectors on queues
+* No browsing or selectors on queues
```
Comment (grs) [inline documentation/modules/con-standard-address-space.adoc]: I'd possibly leave these as two separate items ('no selectors on queues' ''no browsing of queues '). Not a big issue though if you prefer this form.

File: documentation/modules/con-standard-address-space.adoc [hunk 2]
```diff
@@ -4,11 +4,11 @@
 
 [id='con-standard-address-space-{context}']
 = Standard Address Space
-The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, it has the following limitations: 
+The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, the standard address space has the following limitations:
```
Comment (grs) [inline documentation/modules/con-standard-address-space.adoc]: I know you didn't add this but 'The implementation of a storage unit is hidden from the client and the routers with a well-defined API.' seems odd to me. What does that mean? @lulf @rgodfrey ?

File: documentation/modules/con-standard-address-space.adoc (hunk 1 above)
Comment (jenmalloy) [reply documentation/modules/con-standard-address-space.adoc]: Nope, that was a typo on my part. Will fix and keep them as two separate items.

File: documentation/modules/con-standard-address-space.adoc (hunk 2 above)
Comment (lulf) [reply documentation/modules/con-standard-address-space.adoc]: @grs I think it is a bit too abstract :) How about replacing storage units with broker and reword to something like "Clients connect to a message router, which forwards messages to/from one or more message brokers" ?

Comment (grs) [reply documentation/modules/con-standard-address-space.adoc]: Yes, that is easier to understand.

Comment (jenmalloy) [reply documentation/modules/con-standard-address-space.adoc]: @grs, @lulf, Thanks, I'll edit this to reflect the new verbiage.

**Pull request merged**